gitlabform/processors/project/__init__.py

```diff
@@ -14,6 +14,9 @@
 from gitlabform.processors.project.files_processor import FilesProcessor
 from gitlabform.processors.project.hooks_processor import HooksProcessor
 from gitlabform.processors.project.members_processor import MembersProcessor
+from gitlabform.processors.project.project_job_token_scope_processor import (
+    ProjectJobTokenScopeProcessor,
+)
 from gitlabform.processors.project.project_processor import ProjectProcessor
 from gitlabform.processors.project.project_push_rules_processor import (
     ProjectPushRulesProcessor,
```

Comment:
Looks like the pattern for the processor names don't include the `project` prefix; except  a few handful of processors. I'm guessing those somehow failed to follow the convention/pattern. What do you think about renaming `project_job_token_scope_processor` to `job_token_scope_processor`?

Reply:
@amimas yeah I saw that, when developing these and the docs, was wondering about a situation where Gitlab add job token scope to things other that `Project` I could definetly see it happening from a security and control perspective.

Happy to rename now and revisit if that comes to pass if you'd like?